docs/python_support/python_support.md

```diff
@@ -0,0 +1,333 @@
+In that case your Dockerfile may look like:
+
+```dockerfile
+FROM openvino/model_server:latest
+USER root
+ENV LD_LIBRARY_PATH=/ovms/lib
+ENV PYTHONPATH=/ovms/lib/python
+RUN apt update && apt install -y python3-pip git
+RUN pip3 install numpy
+ENTRYPOINT [ `/ovms/bin/ovms` ]
+```
+
+TODO: Here we could provide make target with requirements location as a parameter.
+
+### Building Model Server From Source
+In above section you use public Docker image. In case you want to build it from source do the following:
+
+```bash
+git clone https://github.com/openvinotoolkit/model_server.git
+cd model_server
+make docker_build MEDIAPIPE_DISABLE=0 PYTHON_DISABLE=0 OV_USE_BINARY=1 RUN_TESTS=0
+cd ..
+```
+Resulting Docker image can be extended with additional layers just as the public one.
+
+## Python Servable
+
+When deploying Python servable, Model Server expects Python file with an `OvmsPythonModel` class implemented:
+
+```python
+class OvmsPythonModel:
+
+    def initialize(self, kwargs):
+        """
+        `initialize` is called when model server loads graph definition. It allows to initialize and maintain state between subsequent execute() calls and even graph instances. 
+        For gRPC unary, graphs are recreated per request. 
+        For gRPC streaming, there can be multiple graph instances existing at the same time. 
+        OvmsPythonModel object is initialized with this method and then shared between all graph instances. Implementing this function is required (should it be?).
+
+        Parameters:
+        -----------
+        kwargs : dict
+            Available arguments:
+            * node_name: string
+                Name of the node in the graph
+            * input_streams: list of strings
+                List of input stream names defined for the node in graph configuration
+            * output_streams: list of strings
+                List of output stream names defined for the node in graph configuration
+        -----------
+        """
+        print("Running initialize...")
+
+    def execute(self, inputs):
+        """
+        `execute` is called in `Process` method of PythonExecutorCalculator which in turn is called by Mediapipe framework. How Mediapipe calls `Process` method for the node depends on the configuration and the two configurations supported by PythonExecutorCalculator are:
+        
+        * Regular: `execute` runs every time the node receives inputs. Produces one set of outputs per one set of inputs. For unary endpoints it's the only possible configuration.
+
+        * Generative: `execute` runs multiple times for single inputs set. Produces multiple sets of outputs over time per single set of inputs. Works only with streaming endpoints. 
```

Comment:
Give hint that it is done by yielding instead of return? Is it really true that "execute" runs "multiple times"? In practice, it only runs once, but continues, right?

Reply:
Generative executes N times when N is number of input streams?
We could steer this behavior with different input_streams handler correct? If so maybe it's worth to mention here or direct to documentation about this?

Reply:
@dkalinowski There are more details below in the section with description, outside of this sample

@atobiszei  No, generative runs N times per just one complete set of inputs. In description section below there are links to `Execution modes` section that describes it further. About handler - calculator works with sync set input handler for generative mode. If user changes that, we don't guarantee it will work as expected.